docs/pagination.md

```diff
@@ -219,10 +219,10 @@ class ShowPosts extends Component
 
 The previous hooks only apply to the default paginator. If you are using a named paginator, you must define the methods using the paginator's name.
 
-For example, below is an example of what a hook for a paginator named `invoices-page` would look like:
```

Comment:
I would revert this, as I think the plural is better here, as the paginator is for a collection of invoices, not a single invoice.

Reply:
There was an inconsitency with the code above. I've changed all to invoices-page.